src/index.html

```diff
@@ -11,6 +11,41 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form class="form"
+     action="#"
+     method="post"
+     data-qa="big"
+     name="bigForm"
+     >
+      <lebel class="searchBar">
+        <img
+          src="images/Search.svg"
+          alt="Search image"
+          class="searchBar__img searchBar__img--big"
+          >
+        <input type="text"
```

Comment:
check form and input tags

Reply:
not fixed

Reply:
I'm sorry but I don't understand what needs to be done here